refinery/tool_manager/tests.py

```diff
@@ -1281,6 +1281,27 @@ def test_load_tools_error_message_yields_vis_registry_info(
                 context.exception.message
             )
 
+    def test_load_tools_command_error_if_branch_and_workflows_specified(self):
+        with self.assertRaises(CommandError):
+            call_command("load_tools", workflows=True,
+                         branch="fake-branch-name")
```

Comment:
And in a similar vein, if we were using more of the built-in machinery, I don't think we'd need explicit checks on their error handling.